harness/determined/cli/xperiment.py

```diff
@@ -0,0 +1,60 @@
+import json
+from argparse import Namespace
+from typing import Any, List
+
+import yaml
+
+from determined.common.api.fapi import auth_required, experiments_api, to_json
+from determined.common.declarative_argparse import Arg, Cmd
+
+# from determined.common.api.fastapi_client.models import V1LoginRequest
+
+# import determined.common.api.swagger as swg
+# import determined.common.api.swagger_client.models as models
+# @swg.auth_required
+# def list(_: Namespace) -> None:
+#     # type info shows for some language servers but not all since the generated code
+#     # is using legacy docstring types
+#     api_response = swg.experiment_api.determined_get_experiments(resource_pools=["default"])
+#     print(api_response.experiments)
+
+
+@auth_required
+def list(args: Namespace) -> None:
+    response = experiments_api.determined_get_experiments(limit=5)
```

Comment:
why do all these methods have `determined` in them?

Reply:
I think they come from operationids in the swagger spec file.
![image](https://user-images.githubusercontent.com/12127420/136117742-5c5731aa-2d08-4df7-a3ef-52f2401e2275.png)

not sure where those come from but we should be able to modify them. they appear in the name of the operation here as well https://docs.determined.ai/latest/rest-api/#/Experiments/Determined_GetExperiments